src/sentry/snuba/metrics/extraction.py

```diff
@@ -683,14 +688,41 @@ def _get_args_support(fields: Sequence[str], used_in_function: str | None = None
     if len(fields) == 0:
         return SupportedBy.both()
 
-    if used_in_function == "apdex":
-        # apdex can have two variations, either apdex() or apdex(value).
-        return SupportedBy(on_demand_metrics=True, standard_metrics=False)
+    if support := _configurable_aggregates_supported_by(fields, used_in_function):
+        return support
 
     arg = fields[0]
     return _get_field_support(arg)
 
 
+def _configurable_aggregates_supported_by(
+    fields: Sequence[str], used_in_function: str | None = ""
+) -> SupportedBy | None:
+    supported_by = None
+    # All usage of apdex is on-demand until we can confirm that apdex(300) can be used on alerts
+    # with standard metrics. For details, see test_get_metric_extraction_config_with_no_spec in
+    # tests/sentry/relay/config/test_metric_extraction.py
+    # Remove this block and update tests once confirmed.
+    if used_in_function == "apdex":
+        return SupportedBy(on_demand_metrics=True, standard_metrics=False)
+
+    if used_in_function in get_configurable_aggregates():
```

Comment:
This block is the one that says `user_misery(300)` is not supported by both. If I remove it, it would go back to the original behaviour:
```diff
diff --git a/src/sentry/snuba/metrics/extraction.py b/src/sentry/snuba/metrics/extraction.py
index 5d614e3fb2..bbcca23f7d 100644
--- a/src/sentry/snuba/metrics/extraction.py
+++ b/src/sentry/snuba/metrics/extraction.py
@@ -706,20 +706,6 @@ def _configurable_aggregates_supported_by(
     if used_in_function == "apdex":
         return SupportedBy(on_demand_metrics=True, standard_metrics=False)

-    if used_in_function in get_configurable_aggregates():
-        value = fields[0]
-        if used_in_function == "count_miserable":
-            value = fields[1]
-            # XXX: Remove this line once we support count_miserable
-            return SupportedBy(on_demand_metrics=False, standard_metrics=True)
-
-        supported_by = (
-            SupportedBy.both()
-            # XXX: Note that a customer can change the default in their project settings
-            if value == str(DEFAULT_PROJECT_THRESHOLD)
-            else SupportedBy(on_demand_metrics=True, standard_metrics=False)
-        )
-
     return supported_by


diff --git a/tests/sentry/snuba/metrics/test_extraction.py b/tests/sentry/snuba/metrics/test_extraction.py
index ddf37f39c9..1bf1776928 100644
--- a/tests/sentry/snuba/metrics/test_extraction.py
+++ b/tests/sentry/snuba/metrics/test_extraction.py
@@ -69,7 +69,7 @@ def test_equality_of_specs(default_project) -> None:
         # XXX: For now, forcing it to be supported by on-demand
         ("apdex(300)", "", True),
         ("apdex(3000)", "", True),
-        ("user_misery(300)", "", False),
+        ("user_misery(300)", "", True),
         ("user_misery(3000)", "", True),
         ("count_miserable(user,300)", "", False),
         # XXX: We need to add support count_miserable in on-demand
```